api/tests/opentrons/protocol_engine/state/test_geometry_view.py

```diff
@@ -2624,3 +2627,89 @@ def test_get_offset_fails_with_off_deck_labware(
     labware_store.handle_action(action)
     offset_location = subject.get_offset_location("labware-id-1")
     assert offset_location is None
+
+
+@pytest.mark.parametrize("frustum", [f for f in RECTANGULAR_TEST_EXAMPLES])
```

Comment:
```suggestion
@pytest.mark.parametrize("frustum", RECTANGULAR_TEST_EXAMPLES)
```
don't need the list comprehension here